src/PKSim.Core/Services/EventBuildingBlockCreator.cs

```diff
@@ -134,36 +134,30 @@ private void createNonApplicationEvents()
 
       private void createApplications(IReadOnlyList<CompoundProperties> compoundPropertiesList)
       {
-         // create ApplicationSet-EventGroup
-         var applicationSet = _objectBaseFactory.Create<EventGroupBuilder>()
-            .WithName(Constants.APPLICATIONS)
-            .WithIcon(Constants.APPLICATIONS);
-
-         applicationSet.SourceCriteria.Add(new MatchTagCondition(Constants.ROOT_CONTAINER_TAG));
-
-         compoundPropertiesList.Each(cp => addProtocol(applicationSet, cp));
-
-         _eventGroupBuildingBlock.Add(applicationSet);
+         compoundPropertiesList.Each(addProtocol);
       }
 
-      private void addProtocol(EventGroupBuilder applicationSet, CompoundProperties compoundProperties)
+      private void addProtocol(CompoundProperties compoundProperties)
       {
          var protocol = compoundProperties.ProtocolProperties.Protocol;
          if (protocol == null)
             return;
 
-         var protocolEventGroup = _objectBaseFactory.Create<EventGroupBuilder>()
-            .WithName(protocol.Name)
-            .WithParentContainer(applicationSet);
+         var eventGroup = _objectBaseFactory.Create<EventGroupBuilder>()
+            .WithName(protocol.Name);
+
+         eventGroup.SourceCriteria.Add(new MatchTagCondition(CoreConstants.Tags.EVENTS));
```

Comment:
@msevestre 
Should we add a check during the simulation creation, that there are no other containers named "Events" in the container structure except the one on the top-level?
Otherwise, if somebody defines a container "Events" elsewhere - we will suddenly get too many applications created.

Reply:
How do you create a container name events? 
We had the same problem before with normal events.

Reply:
User could create an Events-Container everywhere - I just didn't think about it before :)
Maybe we should just extend our container naming rules
![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/7ff8d7c3-cac6-4778-9b58-3fdb3c2f4f39)

Reply:
Ok but the code that I fixed is in PKSim
At most , this should be done in MoBi but not in PKSim

Reply:
Yep, that's in MoBi.
Not a high prio anyway